rust/numaflow/src/main.rs

```diff
@@ -49,13 +51,13 @@ async fn run(cli: clap::Command) -> Result<(), Box<dyn Error>> {
 
     match cli_matches.subcommand() {
         Some(("monitor", _)) => {
-            info!("Starting monitor");
+            info!(VERSION_INFO, "Starting monitor");
```

Comment:
let's do `VERSION_INFO` on top, just once as single line, not with every command.